apps/docs/app/global.css

```
@@ -783,3 +783,150 @@ body {
+  background: var(--nb-black);
+  color: var(--nb-cream);
+}
+
+.nb-stepper-step-number {
+  width: 1.25rem;
+  height: 1.25rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 0.6rem;
+  border: 1.5px solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)